ui/src/reducers/index.js

```diff
@@ -1,34 +1,63 @@
-import { APPS_FAILURE, APPS_SUCCESS, APPS_REQUEST, APPS_SORT, TOGGLE_HEADER_DROPDOWN } from '../actions/types.js';
+import {
+  APPS_FAILURE,
+  APPS_SUCCESS,
+  APPS_REQUEST,
+  APPS_SORT,
+  TOGGLE_HEADER_DROPDOWN,
+  APP_DETAILS_SORT
+} from '../actions/types.js';
 
 import { SortByDirection, sortable } from '@patternfly/react-table';
 
-const columns = [
-  { title: 'App Name', transforms: [ sortable ] },
-  { title: 'App ID', transforms: [ sortable ] },
-  { title: 'Deployed Versions', transforms: [ sortable ] },
-  { title: 'Current Installs', transforms: [ sortable ] },
-  { title: 'Launches', transforms: [ sortable ] }
-];
-
-const apps = { rows: [], data: {} };
-
-const sortBy = { direction: SortByDirection.asc, index: 0 };
-
 const initialState = {
-  apps: apps,
-  sortBy: sortBy,
-  columns: columns,
+  apps: { rows: [], data: {} },
+  sortBy: { direction: SortByDirection.asc, index: 0 },
+  columns: [
+    { title: 'App Name', transforms: [ sortable ] },
+    { title: 'App ID', transforms: [ sortable ] },
+    { title: 'Deployed Versions', transforms: [ sortable ] },
+    { title: 'Current Installs', transforms: [ sortable ] },
+    { title: 'Launches', transforms: [ sortable ] }
+  ],
+  appDetailRows: [
+    [ 'v1.0', '25', '321', '2019-01-01 09:55:10', true, 'Deprecated. Please upgrade to latest version' ],
```

Comment:
@laurafitzgerald @craicoverflow going to leave this test data in place until PR by @craicoverflow to retrieve data from API call is in place. Can remove now if preferred, but may make testing more difficult until API is pulling real data

Reply:
@damienomurchu I don't think we should merge this. Seeding the database should give you enough data for testing. If not we should create a JIRA to allow for additional data to be seeded.

Reply:
removed now, so resolving